test/rekt/features/sinkbinding/feature.go

```diff
@@ -103,3 +117,16 @@ func SinkBindingV1Job() *feature.Feature {
 
 	return f
 }
+
+// AsTrackerReference returns a tracker.Reference for a Job without namespace.
+func AsTrackerReference(name string) *tracker.Reference {
+	return &tracker.Reference{
+		Kind:       "Job",
+		APIVersion: "batch/v1",
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": name,
+			},
+		},
+	}
+}
```

Comment:
Not sure if I understood you correctly, but wouldn't `kantive.dev/reconciler-test/pkg/resources/job` be a better location (could be done as a follow up too)?

Reply:
A `tracker.Reference` object is only used in the sinkbinding package why would this function be in a different package when other packages won't use it?

Reply:
why is it better?

Reply:
I was thinking about it for consistency, because some other packages in reconciler-test have it too (e.g. `resources/svc`, `pkg/resources/deployment`).

Reply:
But I am fine with this PR, as it is some kind of cleanup :)

Reply:
I see, hmm, I don't see a usage of svc.AsTrackerReference functions in eventing? Maybe they are not needed?

`deployment.AsTrackerReference` is only used for SinkBinding testing

Reply:
Let me think about this, maybe I will propose something next